aip/0166.md

```diff
@@ -0,0 +1,78 @@
+---
+aip:
+  id: 166
+  state: reviewing
+  created: 2020-06-24
+permalink: /166
+redirect_from:
+  - /0166
+---
+
+# Hierarchically-derived resources
+
+Services occasionally have resources where the resource's effective values are
+determined by consulting both configuration on the resource as well as
+configuration elsewhere, such as a parent in the hierarchy, and "flattening"
+them together. This is most common for configuration resources, which could be
+applied against a resource or a container.
```

Comment:
I don't understand what "applied against a resource or a container" means here.

Reply:
Fixed, is that better?